namui-cli/src/util/get_namui_bundle_list/get_namui_bundle_list.rs

```diff
@@ -0,0 +1,22 @@
+use super::{
```

Comment:
I think directory name should be bundle_list, not get_namui_bundle_list..?

`get_namui_bundle_list/get_namui_bundle_list.rs` --> `bundle_list/get_namui_bundle_list`